ably/auth.go

```diff
@@ -33,6 +42,18 @@ func (c *Capability) UnmarshalJSON(p []byte) error {
 	return json.Unmarshal([]byte(s), (*map[string][]string)(c))
 }
 
+func (c Capability) MarshalMsgpack() ([]byte, error) {
+	return msgpack.Marshal(c.String())
+}
+
+func (c *Capability) UnmarshalMsgpack(p []byte) error {
+	var s string
+	if err := msgpack.Unmarshal(p, &s); err != nil {
+		return err
+	}
+	return json.Unmarshal([]byte(s), (*map[string][]string)(c))
+}
+
```

Comment:
Ok, I think it's worth adding a comment that the capabilities are sent as JSON msgpack strings rather than msgpack maps, hence the need for the custom Marshal methods.